bin/config.toml

```diff
@@ -100,7 +100,7 @@ buffer_size = 16393
 # defines how the TLS protocol will be handled by sozu. Possible values
 # are "openssl" or "rustls". The "openssl" option will only work if sozu
 # was built with the "use-openssl" feature.
-tls_provider = "rustls"
+tls_provider = "openssl"
```

Comment:
maybe reset this to `rustls` so as not to change the default behaviour of sozu out-of-the-box.